src/locales/ko/mystery-encounters/mysterious-chest-dialogue.json

```
@@ -1,23 +1,23 @@
 {
-  "intro": "You found...@d{32} a chest?",
-  "title": "The Mysterious Chest",
-  "description": "A beautifully ornamented chest stands on the ground. There must be something good inside... right?",
-  "query": "Will you open it?",
+  "intro": "이건...@d{32} 뭐지?",
+  "title": "수수께끼 상자",
+  "description": "아름답게 장식된 상자가 바닥에 놓여있습니다. 보통 이럴 땐, 안에 뭔가 좋은 것이 있는 것이 인지상정...\n그렇지 않을까요?",
```

```suggestion
  "description": "아름답게 장식된 상자가 바닥에 놓여 있습니다. 보통 이럴 땐, 안에 뭔가 좋은 게 들었다고 생각하는 것이 인지상정…\n그렇지 않을까요?",
```
"인지상정"은 현상보다는 어떤 행위나 마음 같은 것에 사용해야 해서 약간 고쳐 보았습니다.